plugins/knife/clearwater-dns-records.rb

```diff
@@ -295,6 +295,12 @@ def dns_records
         :value => ipv4s_local_site(find_active_nodes("vellum"), i),
         :ttl   => "60"
       },
+
+      "chronos-site#{i}" => {
+        :type  => "A",
+        :value => ipv4s_local_site(find_active_nodes("vellum"), i),
+        :ttl   => "60"
+      },
```

Comment:
Now that we have chronos hostnames, should we be updating the shared config to use them?
https://github.com/Metaswitch/chef/blob/split-clusters/cookbooks/clearwater/recipes/shared_config.rb#L77

It doesn't actually make a difference though because they both resolve to the same thing.

Actually why do we need this?  If it's for the chronos callback URI, don't we need to be setting that in shared config too?